src/libs/notification/example/README.md

```diff
@@ -1,28 +0,0 @@
-# Example Applications for Notifications
```

Comment:
Why is this README removed?

Btw. the  src/libs/notification/example is quite hidden, maybe you can add links to it also at other places.

Reply:
I forgot to delete the folder - I moved them to /examples. 

About the README: The existing examples didn't have one so I removed it. On second thought I should add instructions about expected plugins to the examples (i.e. do a `printf()`).

Reply:
Here is an updated example: https://github.com/ElektraInitiative/libelektra/pull/1850/files#diff-98f467ea2a20eaed5714de95cc0d4142R193